source/Cute/Commands/Content/ContentTranslateCommand.cs

```diff
@@ -178,6 +195,8 @@ await ProgressBars.Instance()
 
                         var defaultLocaleFieldNames = flatEntry.Keys.Where(k => k.Contains($".{defaultLocale.Code}")).ToArray();
 
+                        Dictionary<string, Task<(string?, bool)>> taskMap = new Dictionary<string, Task<(string?, bool)>>();
+
```

Comment:
There's no limit on concurrent translations, which could overwhelm the translation service or consume too many resources
Adding a semaphore or using ParallelOptions.MaxDegreeOfParallelism would help